src/dumpsxiso/main.cpp

```diff
@@ -778,6 +875,11 @@ void ParseISO(cd::IsoReader& reader) {
 
     size_t numEntries = pathTable.ReadPathTable(&reader, descriptor.pathTable1Offs);
 
+		std::vector<cd::IsoPathTable::Entry> sorted(pathTable.pathTableList);
+		std::sort(sorted.begin(), sorted.end(), [](auto& a, auto& b) {
```

Comment:
```suggestion
		std::sort(sorted.begin(), sorted.end(), [](const auto& a, const auto& b) {
```

AFAIK non-const predicate parameters might not compile on all compilers, better safe than sorry.